### PR TITLE
feat(ui): remove gitea

### PR DESCRIPTION
**Pull request by alexrodfe**
## Motivation and Context

Remove all Gitea components from UI.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] I have created tests for my code changes, and the tests are passing.
- [x] I have executed the pre-commit hooks locally.
- [x] I have updated the documentation accordingly.
- [ ] The commit message follows our guidelines: https://github.com/konstellation-io/kdl-server/blob/main/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: app/ui/public/config.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "SERVER_URL": "http://localhost:4000",
   "KNOWLEDGE_GALAXY_ENABLED": false,
   "KG_SERVER_URL": "http://localhost:4001",
-  "GITEA_URL": "https://gitea.local",
```
Comment (igzjaviergil) [inline app/ui/public/config.json:6]: this change may affect helm-chart, please double check with @ialejandro

File: app/ui/src/Pages/Project/panels/ProjectSettings/components/TabGit/TabGit.tsx [hunk 2]
```diff
@@ -31,20 +31,8 @@ function TabGit({ project }: Props) {
...
-  function openExternalRepoSettings() {
-    const projectGiteaUrl = new URL(toolUrls.gitea);
-    projectGiteaUrl.pathname += '/settings';
-
-    window.open(projectGiteaUrl.toString());
-  }
-
   return (
     <div className={styles.container} data-testid="tabGit">
-      {isAdmin && isExternal && (
```
Comment (igzjaviergil) [inline app/ui/src/Pages/Project/panels/ProjectSettings/components/TabGit/TabGit.tsx:43]: removing entire block of code avoid possibility to edit project, is that correct?

**Review by igzjaviergil — CHANGES_REQUESTED** [review]

File: app/ui/public/config.json (hunk 1 above)
Comment (ialejandro) [reply app/ui/public/config.json:6]: I'll remove from my side too, thx @igzjaviergil

File: app/ui/src/Pages/Project/panels/ProjectSettings/components/TabGit/TabGit.tsx (hunk 2 above)
Comment (alexrodfe) [reply app/ui/src/Pages/Project/panels/ProjectSettings/components/TabGit/TabGit.tsx:43]: I have my own set of doubts as well, however check that the URL the selector navigates to is a Gitea one, and we are deprecating all Gitea. So I believe this service won't be available and there will be no place to go to edit

Comment (igzjaviergil) [reply app/ui/src/Pages/Project/panels/ProjectSettings/components/TabGit/TabGit.tsx:43]: true indeed

**Review by igzjaviergil — APPROVED** [review]

**Review by kafkaphoenix — APPROVED** [review]

**Pull request merged**